src/main/java/hilfling/backend/hilfing/controllers/CategoryController.java

```diff
@@ -0,0 +1,26 @@
+package hilfling.backend.hilfing.controllers;
+
+import hilfling.backend.hilfing.model.Category;
+import hilfling.backend.hilfing.service.CategoryService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1/category")
+public class CategoryController {
+    @Autowired
+    private CategoryService categoryService;
+
+    @GetMapping()
+    public List<Category> getCategories() {
+        return categoryService.getCategories();
```

Comment:
Endre til getAllCategories

Legg til endpoint for getCategory

Reply:
Fixed!